### PR TITLE
Feat/investor feed api responses

### PR DESCRIPTION
**Pull request by cjmaret**
# Describe your changes 

At this PR I:
1. created GET /startups API in the startups route
2. created PATCH /investor/:id API in the users route

I've also had some issues with inadvertently changing both `package-lock`s when i run npm install after a git pull. In the `backend`, only the name key in the packages object was removed, and i replaced that. In the `frontend`, some other changes occurred, and I didn't know what to do about them. These alter the `package-lock`s from how they look in the remote repository. I have no idea why, and I haven't been able to figure out how to fix this. 

If the reviewers can run the code and make sure everything is still working properly (as it is in my local repo), that would be great ⭐ 

# I follow best practices 

- [ ] My code is covered by unit tests 
- [ ] This PR is less than [300 lines of code](https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests)
- [ ] My code [smells good](https://en.wikipedia.org/wiki/Code_smell) 
- [ ] I have tested my PR locally and proven it works. 
- [ ] There is [no copy-paste](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

### CODE REVIEW

**Review by McCambley — CHANGES_REQUESTED** [review]
Hey Colin,
A few small changes needed before merging. A few other changes suggested that aren't necessarily needed. One thing I didn't comment on here that I'm curious about is whether it would make sense to change `user` to `investor` in the file tree in order to keep things consistent. Doesn't need to happen here, maybe later if it seems like it needs to happen. If you fix the response for the patch I'll merge this.

File: backend/routes/users.js [hunk 1]
```diff
@@ -16,7 +16,31 @@ router.get('/api/investors/:id', (req, res) => {
                             'growth-speed']
      }
    })
-})
+});
+
+router.patch('/api/investor/:id', (req, res) => {
```
[the remark this answers is not in the document]
Comment (cjmaret) [reply backend/routes/users.js:21]: ah! great, someone else had done the investor GET, and i didnt realize about the route name

File: backend/routes/users.js [hunk 2]
```diff
@@ -16,7 +16,31 @@ router.get('/api/investors/:id', (req, res) => {
...
+  res.send({
```
[the remark this answers is not in the document]
Comment (cjmaret) [reply backend/routes/users.js:22]: ah! good catch

File: backend/routes/startups.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+router.get('/api/startups', (req, res) => {
+  res.send({
+    'response': {
+      '_id': '7',
+      'investorId': '8',
+      'createdAt': '2021-10-28T00:52:36Z',
+      'name': 'Great Startup',
+      'description': 'The best way to do the thing',
+      'url': 'www.greatstartup.com',
+      'pitchUrl': 'http://youtube.com/greatpitch',
```
[the remark this answers is not in the document]
Comment (cjmaret) [reply backend/routes/startups.js:14]: sure! ill throw that in now

**Comment by cjmaret** [comment]
i've just changed the path from 'users' to 'investors', on top of all the other changes! thanks @jake

**Review by McCambley — APPROVED** [review]
Nice! Aaaaaaapproved ✅

**Pull request merged**